_posts/en/newsletters/2020-07-22-newsletter.md

```diff
@@ -0,0 +1,185 @@
+---
+title: 'Bitcoin Optech Newsletter #107'
+permalink: /en/newsletters/2020/07/22/
+name: 2020-07-22-newsletter
+slug: 2020-07-22-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to several discussions about activating
+taproot and summarizes a proposed update to BIP173 bech32 addresses.
```

Comment:
maybe link BIP173 to https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki here or on line 46 below

Reply:
Adding below.  Just FYI, we have to be careful about adding links in the first paragraph because that text is used for the excerpt on other pages (e.g. https://bitcoinops.org/en/newsletters/ ), which can often break links.  To avoid that, I have a soft rule of never putting links in the lede.

Reply:
> soft rule of never putting links in the lede.

Sounds good, thanks @harding.